src/Workspaces/SharedUtilitiesAndExtensions/Compiler/Core/Formatting/ISyntaxFormattingService.cs

```diff
@@ -17,16 +17,19 @@
 
 namespace Microsoft.CodeAnalysis.Formatting
 {
-    internal interface ISyntaxFormattingService
-#if !CODE_STYLE
-        : ILanguageService
-#endif
+    internal interface ISyntaxFormatting
     {
         SyntaxFormattingOptions GetFormattingOptions(AnalyzerConfigOptions options);
         IEnumerable<AbstractFormattingRule> GetDefaultFormattingRules();
         IFormattingResult GetFormattingResult(SyntaxNode node, IEnumerable<TextSpan>? spans, SyntaxFormattingOptions options, IEnumerable<AbstractFormattingRule>? rules, CancellationToken cancellationToken);
     }
 
+#if !CODE_STYLE
+    internal interface ISyntaxFormattingService : ISyntaxFormatting, ILanguageService
```

Comment:
Move to Workspaces layer.

Reply:
yes.  that makes sense.  will do in followup!